docs/guides/advanced/alternative-clients.md

```diff
@@ -44,3 +44,70 @@ exporter = OTLPSpanExporter(
     headers={'Authorization': 'your-write-token'},
 )
 ```
+
+
+## Example with Rust
+
+First, set up a new Cargo project:
+
+```sh
+cargo new --bin otel-example && cd otel-example
```

Comment:
Is the command needed if you create `Cargo.toml` manually?

Reply:
No, but it doesn't hurt to help align them with the current structure?